tasks/analysis.py

```diff
@@ -435,11 +436,29 @@ def process_impact_result(self, impact_result, analysis_id):
             # pass on template path according to headless service
             custom_template_path = headless_template_path
 
+        # define the layer order of the map report
+        layer_order = (
+            list(settings.LAYER_ORDER) if settings.LAYER_ORDER else None)
+        if layer_order:
+            layer_source = {
```

Comment:
This will miss the layer styles (because it's not in QML). But we will handle that later.

Reply:
I took care of it in InaSAFE side IIRC. Let's see if it works.